eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/AsyncHttpRequestManager.java

```diff
@@ -31,18 +35,48 @@ public class AsyncHttpRequestManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(AsyncHttpRequestManager.class);
 
   /**
-   * Default connect timeout for HTTP connections (in milliseconds).
+   * Connection request timeout for HTTP requests (in seconds). This value is the timeout for
+   * requesting a connection from the connection manager, not for the HTTP request connection
+   * itself.
+   *
+   * @since 1.0.0
+   */
+  private static final int CONNECTION_REQUEST_TIMEOUT_SECS = 900;
```

Comment:
900 seconds for a connection request? this seems like a really long amount of time.  30 seconds sound more reasonable.

Reply:
This is not a timeout for the HTTP request/connection itself. It is a timeout where the request can sit in a queue and wait to be performed.

This is necessary to support extremely large Ewon count environments, where, for example, a metadata update can result in hundreds of requests being performed. Due to the throughput limitations of Talk2M/M2Web, all requests cannot be done simultaneously, therefore the connector must have the capacity to queue those requests.

Reply:
During testing, metadata updates can take a large amount of time. Extending this timeout from its default, which is rather short, significantly reduces the possibility that cascading failures can extend metadata updating much longer.

For example:
- Scenario 1: Cxn Request Timeout is Default
  - Metadata and tag update requests are queued for 200 Ewons at once
  - 50 requests are successful. The remaining 150 fail because of a cxn request timeout
  - Due to throughput limitations, the requests simply could not have been started sooner and timed out before they could even be attempted
- Scenario 2: Cxn Request Timeout is Increased (900)
  - Metadata and tag update requests are queue for 200 Ewons at once
  - 200 requests are successful. None fail due to the increased cxn request timeout

Reply:
Should this be renamed to provided a better description of its functionality

Reply:
Would you happen to have any suggestions?
 
I hoped the comment provided context because these timeout variable names align with their counterpart names in the HTTP library itself.

Reply:
Would CONNECTION_REQUEST_QUEUE_TIMEOUT_SECS work?

Reply:
Yes.  Will update

Reply:
Updated